vmdb/app/models/host.rb

```diff
@@ -821,7 +822,7 @@ def self.refresh_ems(host_ids)
   end
 
   def refresh_ems
-    raise "No #{ui_lookup(:table => "ext_management_systems")} or credentials defined" unless self.ext_management_system && self.ext_management_system.authentication_valid?
+    raise "No #{ui_lookup(:table => "ext_management_systems")} or credentials defined" unless self.ext_management_system && self.ext_management_system.has_credentials?
     EmsRefresh.queue_refresh(self)
```

Comment:
Do we really want to queue a refresh if we've detected system is unavailable or we have a bad password?